pkg/network/ebpf/c/tracer-events.h

```diff
@@ -59,10 +60,20 @@ static __always_inline void cleanup_conn(conn_tuple_t *tup) {
     }
 
     cst = bpf_map_lookup_elem(&conn_stats, &(conn.tup));
+    if (!cst && is_udp) {
+        increment_telemetry_count(udp_dropped_conns);
+        return; // nothing to report
+    }
+
     if (cst) {
         conn.conn_stats = *cst;
         bpf_map_delete_elem(&conn_stats, &(conn.tup));
+    } else {
+        // we don't have any stats for the connection,
+        // so cookie is not set, set it here
+        conn.conn_stats.cookie = get_sk_cookie(sk);
```

Comment:
In a follow up commit, would you please at a comment in `conn_stats_ts_t` explaining what the cookie field is?  It's really not clear to a reader how this field is supposed to be used.